apps/common-lib/src/test/scala/com/gu/typerighter/rules/DictionaryTest.scala

```diff
@@ -19,6 +19,22 @@ class DictionaryTest extends AnyFlatSpec with Matchers {
     expected should be(actual)
   }
 
+  "dictionaryXmlToWordList" should "not include strings that only contain whitespace" in {
```

Comment:
Is it worth adding a test for being more specific in which nodes it retrieves words from?

Reply:
Good idea, 6219a18bd17972b8cec69022b08bb028661e79c9.